Resources/Prototypes/_EE/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -0,0 +1,15 @@
+- type: entity
+  name: CentCom disabler
+  parent: [BaseC2ContrabandUnredeemable, WeaponDisabler]
```

```suggestion
  name: CentComm disabler # Frontier: extra M
  parent: [BaseC2ContrabandUnredeemable, WeaponDisabler] # Frontier: add BaseC2ContrabandUnredeemable
```
certified centcomm moment